### ISSUE TITLE
Create a ThreadMessage component

### ISSUE THREAD

**Issue by bosiraphael** [twentyhq/twenty#3293]

**Comment by ARSHIL1804** [comment]
I would like to work on this @bosiraphael

**Comment by bosiraphael** [comment]
Great ! Thank you @ARSHIL1804, I'm assigning you to this issue.

**Comment by ARSHIL1804** [comment]
@bosiraphael  This whole Thread related backend is not ready yet and we need to create UI for now, right?

**Comment by bosiraphael** [comment]
@ARSHIL1804, yes exactly ! We will use mock data for now and connect the backend as soon as it is ready.

**Comment by bosiraphael** [comment]
Hello @ARSHIL1804, thank you for wanting to contribute to twenty. Sorry for the inconvenience but I assigned @thomtrp to this issue since we need this feature soon. Feel free to take any other good first issue :)

**Comment by ARSHIL1804** [comment]
@bosiraphael No issues.
